docs/modules/server.rst

```diff
@@ -23,7 +23,5 @@ It then calls the method linked to said route and returns the payload to the cli
 
 .. currentmodule:: discord.ext.ipc.server
 
-.. autofunction:: route
```

Comment:
How do you expect anybody to know that this is deprecated if you remove it from the documentation!

Reply:
Forgot to add back in after adding back the deprecated methods.